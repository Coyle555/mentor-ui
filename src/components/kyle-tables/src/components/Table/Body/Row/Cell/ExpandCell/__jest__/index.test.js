import React from 'react';
import { ExpandCell } from '../index';
import renderer from 'react-test-renderer';

function onClick() {}

test('Expand row icon is collapsed on render', () => {
	const component = renderer.create(
		<ExpandCell onClick={onClick} />
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});

test('Expand row icon is expanded on render', () => {
	const component = renderer.create(
		<ExpandCell expanded={true} onClick={onClick} />
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
