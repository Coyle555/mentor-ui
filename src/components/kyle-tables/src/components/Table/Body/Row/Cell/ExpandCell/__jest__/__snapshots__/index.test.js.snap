// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expand row icon is collapsed on render 1`] = `
<td
  className="table-expand-icon table-btn-border"
  onClick={[Function]}
>
  <i
    className="fas fa-caret-right fa-2x"
  />
</td>
`;

exports[`Expand row icon is expanded on render 1`] = `
<td
  className="table-expand-icon table-btn-border"
  onClick={[Function]}
>
  <i
    className="fas fa-caret-down fa-2x"
  />
</td>
`;
